src/src_zoh.c

```diff
@@ -6,15 +6,15 @@
 ** file at : https://github.com/libsndfile/libsamplerate/blob/master/COPYING
 */
 
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
```

Comment:
make it consistent: Add an empty line after this block in every file

Reply:
Done.